contracts/ERC7432/NftRolesRegistryVault.sol

```diff
@@ -13,6 +13,11 @@ contract NftRolesRegistryVault is IERC7432 {
         bytes data;
     }
 
+    bytes32[] public allowedRoles = [keccak256('UNIQUE_ROLE')];
+
+    // roleId => isAllowed
+    mapping(bytes32 => bool) public isRoleAllowed;
```

Comment:
**suggestion (non-blocking):** S Since we have a dynamic `isRoleAllowed` mapping, I think `allowedRoles` can be dynamic too and be set in the constructor (like the other one)
``` suggestion
    bytes32[] public allowedRoles;

    // roleId => isAllowed
    mapping(bytes32 => bool) public isRoleAllowed;
```